### PR TITLE
Add issue templates to promote and guide user engagement

### LINKED ISSUE #20: Adding issue templates
To facilitate contribution, I want to add four issue-templates for: 

- developing lessons
- reporting bug
- suggesting a feature for the repository
- asking questions

The pull request is here #21

### PR DESCRIPTION
**Pull request by SarahAlidoost**
This PR adds four templates for issues: 
- bug_report, 
- lesson_contribution, 
- feature_suggestion, 
- q_and_a. 

This will provide guidance and rules of engagement for people with questions or feedback on the ESMValTool tutorial(s). It will help us keep an overview and make sure the issues are actionable.

It closes #20

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/lesson_contribution.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+name: New lesson material
+about: Develop lessons
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+**Short description of the diagnostic**
+Add a short description of the mterial that you would like to add.
```
Comment (JaroCamphuijsen) [inline .github/ISSUE_TEMPLATE/lesson_contribution.md]:
```suggestion
Add a short description of the material that you would like to add.
```

**Comment by SarahAlidoost** [comment]
> Looks good, I fixed one small typo. What do you think @ledm can we merge this??

@ledm I think we can merge this PR.

**Comment by SarahAlidoost** [comment]
> Hey @SarahAlidoost ,
> Thanks for the effort you've put into creating this PR. To help me review this, can you please add a description/explanation either here or in the issue? What problem does this PR solve or what improved functionality does it add?
> 
> If I look at the 8 issues that are currently in the repo, I don't think they really fit into these two categories. Shouldn't we wait a bit longer until the lesson is really starting to be used and we're getting feedback and hence more insight into the types of issues that we might expect for this repo? Or perhaps make just one general template for all types of issues for now, and extend it later to include appropriate categories?

@Peter9192 thanks a lot. 
I updated the description of this PR in its issue here https://github.com/ESMValGroup/tutorial/issues/20. 

Regarding those 8 issues that are not related to either lessons or bug, I made another template which is for adding a new feature to the repository. 
Actually, these issue-templates facilitate the contributions. In [the PR template](https://github.com/ESMValGroup/tutorial/blob/master/.github/PULL_REQUEST_TEMPLATE.md), developers are asked to make an issue before they start contributing. I see your points about getting more insights about the types of issues. For that purpose, this part of the repository is also included in continuous improvements similar to other parts.

**Comment by SarahAlidoost** [comment]
@Peter9192 thank you for your helpful feedback. Now there are four templates in this pull request that cover most of the issues.

File: .github/ISSUE_TEMPLATE/lesson_contribution.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+name: New lesson material
+about: Develop lessons
+title: ''
+labels: enhancement
```
Comment (Peter9192) [inline .github/ISSUE_TEMPLATE/lesson_contribution.md:5]: Shouldn't this label be different from the other one ('suggestion')?

File: .github/ISSUE_TEMPLATE/q_and_a.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+---
+name: Question and answer
+about: Ask your questions
+title: ''
+labels: question
```
Comment (Peter9192) [inline .github/ISSUE_TEMPLATE/q_and_a.md:5]: We now have two labels: 'question' and 'help wanted'. Aren't these the same?

File: .github/ISSUE_TEMPLATE/lesson_contribution.md (hunk 2 above)
Comment (SarahAlidoost) [reply .github/ISSUE_TEMPLATE/lesson_contribution.md]: Thanks, I separated them by adding more labels. Just like the type of issues, the list of the labels can be improved as well.

File: .github/ISSUE_TEMPLATE/q_and_a.md (hunk 3 above)
Comment (SarahAlidoost) [reply .github/ISSUE_TEMPLATE/q_and_a.md:5]: thanks for pointing this out. Yes, they are the same. I should edit the default list of the labels in this repo.

**Pull request merged**